File: TransCoder_tokenized_test_set_functions/java/FIND_THE_MINIMUM_DISTANCE_BETWEEN_TWO_NUMBERS_1.java
int minDist ( int arr [ ], int n, int x, int y ) {
  int i = 0;
  int min_dist = Integer . MAX_VALUE;
  int prev = 0;
  for ( i = 0;
  i < n;
  i ++ ) {
    if ( arr [ i ] == x || arr [ i ] == y ) {
      prev = i;
      break;
    }
  }
  for (;
  i < n;
  i ++ ) {
    if ( arr [ i ] == x || arr [ i ] == y ) {
      if ( arr [ prev ] != arr [ i ] && ( i - prev ) < min_dist ) {
        min_dist = i - prev;
        prev = i;
      }
      else prev = i;
    }
  }
  return min_dist;
}

File: results/py/FIND_THE_MINIMUM_DISTANCE_BETWEEN_TWO_NUMBERS_1.cpp
int f_filled ( int arr [ ] , int n , int x , int y ) {
  int f_filled = INT_MAX ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( arr [ i ] == x || arr [ i ] == y ) {
      prev = i ;
      break ;
    }
  }
  while ( i < n ) {
    if ( arr [ i ] == x || arr [ i ] == y ) {
      if ( arr [ prev ] != arr [ i ] && ( i - prev ) < f_filled ) {
        f_filled = i - prev ;
        prev = i ;
      }
      else {
        prev = i ;
      }
    }
    i ++ ;
  }
  return f_filled ;
}
